licenses.yml

```diff
@@ -71,6 +71,10 @@
   name: Common Public Attribution License Version 1.0
   url: https://spdx.org/licenses/CPAL-1.0.html
 
+- identifier: Elastic-2.0
+  name: Elastic License 2.0
+  url: https://spdx.org/licenses/Elastic-2.0.html
+
```

Comment:
Elastic License is not Free and Open-Source https://spdx.org/licenses/

~~This block should be moved to `licenses-nonfree.yml`~~ Actually it's already there so no need to add it again

Reply:
I removed it.